queries.yaml

```diff
@@ -168,3 +168,7 @@ post_process:
     source_layer: landuse
     label_property_name: label_position
     label_property_value: "yes"
+  TileStache.Goodies.VecTiles.transform.generate_label_features:
```

Comment:
Ah, nice catch.

I'd vote for making it into a list instead. I haven't checked the code, but it seems like that should be a relatively easy change to make.